hwdb.d/60-keyboard.hwdb

```diff
@@ -153,6 +153,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*8930:*
 
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*7750G:*
  KEYBOARD_KEY_e0=!pageup
+ 
```

Comment:
extra space here breaks the build, it needs to be an empty line

`Cannot parse /home/runner/work/systemd/systemd/build/../hwdb.d/60-keyboard.hwdb: Expected stringEnd, found 'e'  (at char 7091), (line:154, col:1)`